File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/service/ServiceActivity.java
package com.jdroid.android.sample.ui.service;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class ServiceActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return ServiceFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/service/ServiceActivity.kt
package com.jdroid.android.sample.ui.service

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class ServiceActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return ServiceFragment::class.java
    }
}
